opensbi-0.9/lib/sbi/sm/enclave.c

```diff
@@ -790,6 +790,8 @@ uintptr_t do_timer_irq(uintptr_t *regs, uintptr_t mcause, uintptr_t mepc)
 		mm_free((void*)(enclave->paddr), enclave->size);
 
 		spin_unlock(&enclave_metadata_lock);
+		csr_read_clear(CSR_MIE, MIP_MTIP);
```

Comment:
It seems the `csr_read_clear` and `csr_read_set` are required for all paths.
Can we put these two codes in one common place, e.g., after the `timer_irq_out:` or after `swap_from_enclave_to_host`?
Instead of putting them on two places now.

Reply:
@Moonquakes